### PR TITLE
Other arch images

### PR DESCRIPTION
**Pull request by gquillar**
Build the images according to the architecture to allow the support of other arches.
The change has been tested to generate ppc64le and s390x images.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -75,14 +82,14 @@ export OPERATOR_NAMESPACE?=openshift-compliance
...
 SDK_VERSION?=v0.18.2
 ifeq ($(OS_NAME), Linux)
-    OPERATOR_SDK_URL=https://github.com/operator-framework/operator-sdk/releases/download/$(SDK_VERSION)/operator-sdk-$(SDK_VERSION)-x86_64-linux-gnu
+    OPERATOR_SDK_URL=https://github.com/operator-framework/operator-sdk/releases/download/$(SDK_VERSION)/operator-sdk-$(SDK_VERSION)-$(ARCH)-linux-gnu
 else ifeq ($(OS_NAME), Darwin)
     OPERATOR_SDK_URL=https://github.com/operator-framework/operator-sdk/releases/download/$(SDK_VERSION)/operator-sdk-$(SDK_VERSION)-x86_64-apple-darwin
 endif
 
-OPM_VERSION=v1.15.2
+OPM_VERSION=v1.20.0
```
Comment (jhrozek) [inline Makefile:90]: @mrogers950 do you suppose this change could bite us while we are using the super outdated operator-sdk as we are?

**Review by jhrozek — COMMENTED** [review]
Mostly looks harmless, unsure if bumping OPM would work well.
That said, I tested `make index-image` and `make test-index-image` and those seem to work fine.

File: Makefile (hunk 1 above)
Comment (mrogers950) [reply Makefile:90]: I think this will be OK.

Comment (jhrozek) [reply Makefile:90]: LGTM, then

**Comment by gquillar** [comment]
@jhrozek still pending though your LGTM. What is expected?

**Review by jhrozek — APPROVED** [review]
/lgtm

**Comment by skuethe** [comment]
looking forward for this PR to hit the (next) release, so we can start using it. Any idea when that will be?

**Pull request merged**